mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -746,10 +744,10 @@ public void startProcessingRequest(MailboxSession session) {
     public List<MailboxPath> list(MailboxSession session) throws MailboxException {
```

Comment:
IMO we could benefit from having that method reacti too.

Reply:
in the next PR, it's already hard to read